src/components/List/List.jsx

```diff
@@ -43,18 +46,27 @@ const List = () => {
       )}
 
       {value && !value.empty && (
-        <ul>
-          {value.docs.map((doc) => (
-            <li key={doc.id}>
-              <input
-                type="checkbox"
-                checked={has24HoursPassed(doc.data().lastDate)}
-                onChange={() => handleCheck(doc.id, doc.data().lastDate)}
-              />
-              {doc.data().name}
-            </li>
-          ))}
-        </ul>
+        <>
+          <input value={filter} onChange={(e) => setFilter(e.target.value)} />
+          {filter.length >= 1 && (
+            <button onClick={() => setFilter('')}>X</button>
```

Comment:
what do you think if create your own function to clear the filter, something like this:

`const clearFilter =()=> setFilter('')`

then, you could use like this: 
`onClick={clearFilter}`


my reason:
https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

Reply:
Got it :D change is done